### PR TITLE
[News] Update SDK and LG

### PR DESCRIPTION
**Pull request by KayMKM**
### Purpose
*What is the context of this pull request? Why is it being done?*
update sdk and lg in news skill

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

**Comment by xieofxie** [comment]
@KayMKM Do you use a different way to open pr? Reviewers are not automatically added for your prs..

**Comment by KayMKM** [comment]
> @KayMKM Do you use a different way to open pr? Reviewers are not automatically added for your prs..

I just open pr in github webpage...

**Comment by KayMKM** [comment]
> shared.lg is not used?

yes... I am not sure should I use wrapper like other skills. because the news skill doesn't using adaptive card, so all the responses are simple. I will remove the shared.lg first. If need to add wrapper I will add it back at that time.

File: skills/csharp/experimental/newsskill/Dialogs/MainDialog.cs [hunk 1]
```diff
@@ -33,19 +30,17 @@ public class MainDialog : ComponentDialog
...
         private Dialog _favoriteTopicsDialog;
-        private LocaleTemplateEngineManager _localeTemplateEngineManager;
-        //private MainResponses _responder = new MainResponses();
+        private LocaleTemplateManager _templateManager;
 
         public MainDialog(
             IServiceProvider serviceProvider,
             IBotTelemetryClient telemetryClient,
-            LocaleTemplateEngineManager localeTemplateEngineManager)
+            LocaleTemplateManager localeTemplateEngineManager)
```
Comment (bobokids) [inline skills/csharp/experimental/newsskill/Dialogs/MainDialog.cs]: templateManager?

Comment (KayMKM) [reply skills/csharp/experimental/newsskill/Dialogs/MainDialog.cs]: fixed

File: skills/csharp/experimental/newsskill/Startup.cs [hunk 2]
```diff
@@ -93,25 +93,25 @@ public void ConfigureServices(IServiceCollection services)
             // Configure localized responses
             var locales = new List<string>() { "en-us", "de-de", "es-es", "fr-fr", "it-it", "zh-cn" };
 
-            var localizedTemplates = new Dictionary<string, List<string>>();
+            var localizedTemplates = new Dictionary<string, string>();
             foreach (var locale in locales)
             {
-                var localeTemplateFiles = new List<string>();
+                string localeTemplateFiles;
```
Comment (ssss141414) [inline skills/csharp/experimental/newsskill/Startup.cs]: Change localeTemplateFiles to localeTemplateFile? Because now it's not a list but only a file...

Comment (KayMKM) [reply skills/csharp/experimental/newsskill/Startup.cs]: fixed

**Pull request merged**